lib/lib-storage/src/Upload.ts

```diff
@@ -276,6 +276,7 @@ export class Upload extends EventEmitter {
       const partResult = await this.client.send(
         new UploadPartCommand({
           ...this.params,
+          ContentLength: partSize || undefined,
```

Comment:
any reason not to always set this to `undefined`?

the body, `dataPart.data` might always be a buffered value with known length